Compiler/BackEnd/BackendInline.mo

```diff
@@ -665,9 +665,9 @@ function: inlineEquationArray
   input BackendDAE.EquationArray inEquationArray;
   input Inline.Functiontuple fns;
   input BackendDAE.Shared iShared;
-  output BackendDAE.EquationArray outEquationArray;
+  output BackendDAE.EquationArray outEquationArray = inEquationArray;
   output BackendDAE.EqSystem outEqs;
```

Comment:
`outEqs := BackendDAEUtil.createEqSystem( BackendVariable.listVar({}), BackendEquation.listEquation({}));`
would be fine, but uses up memory for the 'normal' cases.
Maybe the `else` part of the try should `fail()`?